cherrypy/_cpreqbody.py

```diff
@@ -553,20 +563,6 @@ class Part(Entity):
 
     """A MIME part entity, part of a multipart entity."""
 
-    # "The default character set, which must be assumed in the absence of a
```

Comment:
You've deleted this comment, which based on the quotes leads me to believe it's a reference in an RFC, so it seems relevant.